File: January 19, 2026.py
def compare_energy(calories_burned, watt_hours_used):

    calories_joules = calories_burned * 4184;
    watt_hours_joules = watt_hours_used * 3600;

    return "Workout" if calories_joules > watt_hours_joules else "Devices" if calories_joules < watt_hours_joules else "Equal"

File: January 19, 2026.js
function compareEnergy(caloriesBurned, wattHoursUsed) {

  const caloriesJoules = caloriesBurned * 4184;
  const wattHoursJoules = wattHoursUsed * 3600;

  return caloriesJoules > wattHoursJoules ? "Workout" : caloriesJoules < wattHoursJoules ? "Devices" : "Equal";
}